### PR TITLE
filter feed reblogs

### PR DESCRIPTION
**Pull request by Adesojisouljay**

### CODE REVIEW

File: src/common/pages/entry-index.tsx [hunk 1]
```diff
@@ -24,25 +24,35 @@ interface Props extends PageProps {
...
 
 class EntryIndexPage extends Component<Props, State> {
     state: State = {
         entryList: [],
         promoted: [],
+        noReblog: false,
     };
 
     componentDidMount() {
+        console.log('componentDidMount');
```
Comment (feruzm) [inline src/common/pages/entry-index.tsx:38]: No need this log

File: src/common/components/entry-index-menu-dropdown/index.tsx [hunk 2]
```diff
@@ -100,12 +104,22 @@ export const EntryIndexMenuDropdown = (props: Props) => {
...
+        ]
+  }
+
   const dropDownConfig = {
     history: null,
     label: (
       <div className='tagDropDown'>
         <span className='pl-2' />
-        {tag === "" ? _t('entry-filter.filter-global') : tag === 'my' ? _t('entry-filter.filter-community') : tag === 'today' ? _t('entry-filter.filter-today') : tag === 'week' ? _t('entry-filter.filter-week') : tag === 'month' ? _t('entry-filter.filter-month'): tag === 'year' ? _t('entry-filter.filter-year') : tag === 'all' ? _t('entry-filter.filter-alltime') : tag}
+        {tag === "" ? _t('entry-filter.filter-global') : tag === 'my' ? _t('entry-filter.filter-community') : tag === 'today' ? _t('entry-filter.filter-today') : tag === 'week' ? _t('entry-filter.filter-week') : tag === 'month' ? _t('entry-filter.filter-month'): tag === 'year' ? _t('entry-filter.filter-year') : tag === 'all' ? _t('entry-filter.filter-alltime') : tag === `@${activeUser.username}` ? (props.noReblog === true ? _t('entry-filter.filter-no-reblog') : _t('entry-filter.filter-with-reblog')) : tag}
```
Comment (feruzm) [inline src/common/components/entry-index-menu-dropdown/index.tsx:122]: There is some logical issue here: For example, if I am logged in with @demo but want to see other user's feed (@ecency's feed) /@ecency/feed then I get in options `@ecency` and `Exclude reblogs`. Also looks like when logout we are not showing feed of any user, this is bug, maybe we can fix that here as well. Find out why it is not rendering/recognizing.

**Review by feruzm — CHANGES_REQUESTED** [review]

**Comment by Adesojisouljay** [comment]
This is noted, I will check and fix bug.

Thanks.

**Comment by Adesojisouljay** [comment]
Issue has been resolved on a new branch.

**Pull request closed without merging**